### ISSUE TITLE
[BUG] Linux unix like environments: Absolute paths generates fonts, images, js and styles in the wrong directory 

### ISSUE THREAD

**Issue by claudiow** [compodoc/compodoc#206]
##### **Overview of the issue**

Assuming the following project lay out

```
/home/<user-home>/projects/myprojects/angular-project
                                       |->src
                                       |  |->app
                                       ...
/home/<user-home>/projects/mydocuments/angular-project
```

Running `compodoc src/app -d /home/<user-home>/projects/mydocuments/angular-project` in `/home/<user-home>/projects/myprojects/angular-project`

Generates 
```
/home/<user-home>/projects/myprojects/angular-project
                                       |->src
                                       |  |->app
                                       ...
                                       | - >home/<user-home>/projects/mydocuments/angular-project
                                                                                |->fonts 
                                                                                |->images 
                                                                                |->js 
                                                                                |->styles 
```
and
```
/home/<user-home>/projects/mydocuments/angular-project
                                       |->classes
                                       |->components
                                       |->graph
                                       |->injectables
                                       |->modules
                                       |->coverage,html
                                       |->index.html
                                       |->modules.html
                                       |->overview.html
```

This breaks the generated documentation.

##### **Operating System, Node.js, npm, compodoc version(s)**

OS: Linux 4.4.0-79-generic Ubuntu - x86_64 GNU/Linux
Node: 7.10.0
NPM: 4.2.0
compodoc: 1.0.0-beta.10 

##### **Angular configuration, a `package.json` file in the root folder**

```json
{
  "name": "angular-project",
  "description":"My project",
  "version": "0.0.0",
  "license": "APACHE-2",
  "scripts": {
    "ng": "ng",
    "start": "ng serve",
    "build": "ng build",
    "build-bh": "ng build --base-href=/a2-struts/ -d js",
    "gendocs": "./node_modules/.bin/compodoc -d /home/<user-home>/projects/mydocuments/angular-project",
    "test": "ng test",
    "test-once": "ng test --code-coverage --single-run --browsers PhantomJS --reporters html",
    "lint": "ng lint",
    "e2e": "ng e2e"
  },
  "private": true,
  "dependencies": {
    "@angular/common": "^4.0.0",
    "@angular/compiler": "^4.0.0",
    "@angular/core": "^4.0.0",
    "@angular/forms": "^4.0.0",
    "@angular/http": "^4.0.0",
    "@angular/platform-browser": "^4.0.0",
    "@angular/platform-browser-dynamic": "^4.0.0",
    "@angular/router": "^4.0.0",
    "angular-in-memory-web-api": "0.3.1",
    "classlist.js": "^1.1.20150312",
    "core-js": "^2.4.1",
    "intl": "^1.2.5",
    "rxjs": "^5.3.0",
    "zone.js": "^0.8.4"
  },
  "devDependencies": {
    "@angular/cli": "1.0.6",
    "@angular/compiler-cli": "^4.0.0",
    "@compodoc/compodoc":"1.0.0-beta.10",
    "@types/jasmine": "2.5.38",
    "@types/node": "~6.0.60",
    "angular-ide": "^0.9.19",
    "codelyzer": "~2.0.0",
    "jasmine-core": "~2.5.2",
    "jasmine-spec-reporter": "~3.2.0",
    "karma": "~1.4.1",
    "karma-chrome-launcher": "~2.0.0",
    "karma-phantomjs-launcher": "^1.0.4",
    "karma-cli": "~1.0.1",
    "karma-jasmine": "~1.1.0",
    "karma-jasmine-html-reporter": "^0.2.2",
    "karma-coverage-istanbul-reporter": "^0.2.0",
    "karma-junit-reporter": "^1.2.0",
    "karma-htmlfile-reporter": "~0.3.5",
    "protractor": "~5.1.0",
    "ts-node": "~2.0.0",
    "tslint": "~4.5.0",
    "typedoc": "^0.7.1",
    "typescript": "~2.2.0"
  }
}
```

##### **Compodoc installed globally or locally ?**

Locally

##### **Motivation for or Use Case**

The generated documentation is broken.

##### **Reproduce the error**

Create a simple project with angular cli.

run compodoc with  `-d <some-absolute-path>`

##### Workaround

Use relative paths `compodoc src/app -d "../mydocuments/angular-project"` in `/home/<user-home>/projects/myprojects/angular-project`

**Comment by vogloblinsky** [comment]
@claudiow 
Seems more related to missing / at the end of supplied output directory.
Your examples always breaks because of missing tsconfig.json file parameter.
Next release coming soon this week, let a feedback if it doesn't work fine.

**Comment by claudiow** [comment]
The missing tsconfig.json and final path separator are typos I made when I created the issue, my actual command line looks like this:

`compodoc src/app -p tsconfig.json -d /home/<user-home>/projects/mydocuments/angular-project/`

Missing a final path separator results in adding whatever is after the last path separator to the name of the files in all OS environments.

I was thinking on this as a separate issue as it breaks the contract of the interface. 

According to the documentation: `-d, --output [folder] | =  Where to store the generated documentation`

IMHO the parser of the parameter should normalize and complete whatever the user enters.

By normalizing I mean replace either / or \ with the System PathSeparator and add a path separator to the end if the paramater is missing it.
